source/extensions/access_loggers/http_grpc/grpc_access_log_impl.h

```diff
@@ -24,63 +24,43 @@ namespace HttpGrpc {
 // TODO(mattklein123): Stats
 
 /**
- * Interface for an access log streamer. The streamer deals with threading and sends access logs
- * on the correct stream.
+ * Interface for an access logger. The logger provides abstraction on top of gRPC stream, deals with
+ * reconnects and performs batching.
  */
-class GrpcAccessLogStreamer {
+class GrpcAccessLogger {
 public:
-  virtual ~GrpcAccessLogStreamer() = default;
+  virtual ~GrpcAccessLogger() = default;
 
-  /**
-   * Send an access log.
-   * @param message supplies the access log to send.
-   * @param log_name supplies the name of the log stream to send on.
-   */
-  virtual void send(envoy::service::accesslog::v2::StreamAccessLogsMessage& message,
-                    const std::string& log_name) PURE;
+  virtual void log(envoy::data::accesslog::v2::HTTPAccessLogEntry& entry) PURE;
```

Comment:
This is not your code, but if the message is being passed in by reference so it can be moved, can you pass by `&&` or pass by const reference.